mandatory/3-playing-computer.js

```diff
@@ -30,7 +30,7 @@ console.log(x);
 console.log(a);
 console.log(b);
 
-for (let i = 0; i < 5; ++i) {
+for (let i = 0; i < 5; i++) {
```

Comment:
Here the question wants to tell you the difference between `++i` and `i++`. How this difference will change the output of this for loop?
We can talk about this in our next 1-1 and/or you can search Google to learn more about it :)